src/common/settingsProvider.ts

```diff
@@ -5,10 +5,15 @@ import { SETTINGS_STORE_NAME } from '../utils';
 import { ThemeString } from './webviewEvents';
 
 const SUPPORTED_THEMES = new Map<string, string>([
-  ['Default Light+', 'light'],
-  ['Visual Studio Light', 'light'],
+  ['Default Light+', 'default'],
+  ['Visual Studio Light', 'default'],
   ['Default Dark+', 'dark'],
   ['Visual Studio Dark', 'dark'],
+  ['System Preference', 'systemPreferred'],
```

Comment:
same here, not sure if this code is needed.

Reply:
This line is mandatory. The DevTools don't know "systemPreference" as a valid theme.